Dockerfile

```
@@ -88,7 +88,7 @@ RUN apt -y update \
 #   pydicom: DICOM images
 #
 
-RUN pip --no-cache-dir install --upgrade pip setuptools wheel ipython
+RUN pip --no-cache-dir install --upgrade pip setuptools wheel build ipython
```

I don't believe `build` is necessary here, as it expects a wheel is already build in `dist`
```suggestion
RUN pip --no-cache-dir install --upgrade pip setuptools wheel ipython
```